pyvlx/pyvlx.py

```diff
@@ -39,42 +35,35 @@ def __init__(self, path=None, host=None, password=None, loop=None):
         self.scenes = Scenes(self)
         self.version = None
         self.protocol_version = None
+        self.klf200 = Klf200Gateway(pyvlx=self)
 
     async def connect(self):
         """Connect to KLF 200."""
-        PYVLXLOG.debug("Connecting to KLF 200.")
+        PYVLXLOG.warning("Connecting to KLF 200.")
         await self.connection.connect()
-        login = Login(pyvlx=self, password=self.config.password)
-        await login.do_api_call()
-        if not login.success:
-            raise PyVLXException("Login to KLF 200 failed, check credentials")
-        await self.update_version()
-        await set_utc(pyvlx=self)
+        await self.klf200.password_enter(password=self.config.password)
+        await self.klf200.get_version()
+        await self.klf200.get_protocol_version()
+        PYVLXLOG.warning(
+            "Connected to: %s, protocol version: %s",
+            str(self.klf200.version),
+            str(self.klf200.protocol_version)
+        )
```

Comment:
Would recommend to change it to debug logs in order to avoid unneccesary warnings in HA.

Reply:
set to debug